src/js/main.js

```diff
@@ -311,6 +325,9 @@ Site.Map = {
     // Move the map
     _this.moveMap(newX,newY);
 
+    // Change pattern
+    _this.changePattern();
```

Comment:
Yeah, i think we should manage this *background* functionality in it's own code, perhaps `Site.Background`. This way we keep every part clean and isolated. 
To bound to the events is as easy as:

```
document.querySelector('#map').addEventListener('stoppan', function() { console.log('stop');});
document.querySelector('#map').addEventListener('startpan', function() { console.log('start');});
```

I favor this approach cuz also we need those events to play/stop the videos around the map when they are on/off screen.